en/docs/guides/Provisioning/outbound-provisioning-with-salesforce.md

```diff
@@ -0,0 +1,190 @@
+    configure the Identity Server to Salesforce.  
+    Make a note of the following details as you need them in upcoming
+    configurations.
+
+    1.  Consumer Key
+    2.  Consumer Secret (Click the **Click to reveal** link to view the
+        consumer secret)
+    3.  Callback URL
+
+    !!! info
+        **Consumer Key** : A value used by the consumer to identify itself
+        to Salesforce. Referred to as client\_id in OAuth 2.0.
+
+        **Consumer Secret** : A secret used by the consumer to establish
+        ownership of the consumer key. Referred to as client\_secret in
+        OAuth 2.0.
+
+    ![consumer-secret](../assets/img/guides/connected-app-screen.png) 
+
+6.  Add your connected app to the profile you are going to use. This is
+    necessary as this profile is used when you add users in to
+    Salesforce from the Identity Server.
+
+    !!! note
+        Allow from 2-10 minutes for your changes to take effect on the
+        server before using the connected app.
+    
+    1.  Expand **Users** in the **Administration** section of the left hand panel and click **Profiles**. A list of existing
+        profiles can be viewed.  
+        ![user-profiles](../assets/img/guides/profiles.png) 
+
+    2.  As an example, if you use the profile “Chatter Free User”, click
+        **Edit** and select the connected app you created to configure
+        with the Identity Server using the provided checkbox.  
+         
+        ![select-connected-app](../assets/img/guides/example-app.png) 
+
+    3.  Click **Save**. Make a note of the profile ID (or address URL obtained from the address bar in your browser)
+        of the Chatter Free User profile.
+
+        !!! tip
+            Copy the URL and decode it using a URL decoder like [urldecoder.org]. 
+
+            ![decoder-online](../assets/img/guides/decoder-online.png)
+
+        
+            In this case `             00e2x000001AT3y            ` is your
+            profile ID.
+        
+
+7.  <a name="public"></a>Get the public certificate for Salesforce. Do the following in order
+    to achieve this.
+
+    !!! info 
+        For more information on generating the certificate, see the [Salesforce
+        documentation](https://help.salesforce.com/articleView?id=security_keys_creating.htm&type=0).
+
+    1.  In the left navigation panel, Expand **Security** and click
+        **Certificate and Key Management** or you can search for
+        Certificate and Key Management in the Quick Find search box.  
+
+    2.  Click **Create Self-Signed Certificate**.
+    3.  Enter the **Label** and a **Unique Name** and click **Save**.
+        The certificate is generated.  
+        ![self-signed-certificate](../assets/img/guides/add-certificate.png) 
+    4.  Click the **Download Certificate** button to download the
+        certificate.
+
+## Configure the Identity Server to use email address as the username
```

Comment:
Shall we make this heading shorter --> "Configure using email address as the username"